examples/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS.js

```diff
@@ -0,0 +1,36 @@
+// Import the NLCD RCMAP TRENDS collection.
+var dataset = ee.ImageCollection('USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/TRENDS');
```

Comment:
This is an image - please change to ee.Image and remove the filtering below.  Make sure the full script works in Code Editor.